packages/web-react/src/components/Container/Container.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { applyClassNamePrefix } from '../../utils/classname';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+import { ChildrenProps, StyleProps } from '../../types';
+
+export interface ContainerProps extends ChildrenProps, StyleProps {
+  tag: ElementType;
+}
+
+const defaultProps = {
+  tag: 'div',
+};
+
+export const Container = ({ tag: Tag, className, children, ...restProps }: ContainerProps): JSX.Element => {
```

Comment:
do we not count on a fluid container?

Reply:
There is no `fluid` property now (https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/components/Container/_Container.scss). This is a question for @adamkudrna or @crishpeen, I think.